src/clients/Tester.ts

```diff
@@ -0,0 +1,90 @@
+import { AppClient, inflightURL, InstanceOptions, IOContext } from '@vtex/api'
+
+export interface SpecTestReport {
+  testId: string
+  title: string[]
+  state: string
+  body: string
+  stack?: string
+  error?: string
+}
+
+export interface Screenshot {
+  screenshotId: string
+  name?: string
+  testId: string
+  takenAt: string
+  path: string
+  height: number
+  width: number
+}
+
+export interface SpecReport {
+  state: 'enqueued' | 'running' | 'passed' | 'failed' | 'skipped' | 'error'
+  error?: string
+  report?: {
+    stats: {
+      suites: number
+      tests: number
+      passes: number
+      pending: number
+      skipped: number
+      failures: number
+    }
+    tests: SpecTestReport[]
+    video?: string
+    screenshots: Screenshot[]
+  }
+  logId?: string
+  lastUpdate: number
+}
+
+export interface AppReport {
+  [spec: string]: SpecReport
+}
+
+export interface TestReport {
+  [appId: string]: AppReport
+}
+
+export interface TestOptions {
+  monitoring: boolean
+  integration: boolean
+  authToken?: string
+  appKey?: string
+  appToken?: string
+}
+
+type Spec = string
+
+export interface TestRequest {
+  testId: string
+  options: TestOptions
+  testers: {
+    [tester: string]: {
+      [appId: string]: Spec[]
+    }
+  }
+  requestedAt: number
+}
+
+export class Tester extends AppClient {
+  constructor(context: IOContext, options?: InstanceOptions) {
+    super('vtex.tester-hub@0.x', context, options)
+  }
+
+  public report(testId: string) {
+    return this.http.get<TestReport>(`/_v/report/${testId}`, {
+      inflightKey: inflightURL,
```

Comment:
https://github.com/vtex/toolbelt/pull/769/files?file-filters%5B%5D=.lock&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r397273329

Here I think it makes some sense having the `inflightKey` since every 2 seconds a request is made to this URL. If the previous request isn't finished, they'll receive the same response and will not make two different requests